Blueprints/BlueprintDefinitions/vs2022/CustomRuntimeFunction/template/src/BlueprintBaseName.1/BlueprintBaseName.1.csproj

```diff
@@ -1,12 +1,19 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <LangVersion>latest</LangVersion>
     <AWSProjectType>Lambda</AWSProjectType>
     <AssemblyName>bootstrap</AssemblyName>
     <!-- This property makes the build directory similar to a publish directory and helps the AWS .NET Lambda Mock Test Tool find project dependencies. -->
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+
+    <!-- Enable trimming to reduce package size of self contained publish -->
+    <PublishTrimmed>true</PublishTrimmed>
```

Comment:
curious, how will this impact on symbols?

Reply:
Can you give more details on what you mean?

Reply:
I mean debug symbol, does it impact debugging abilities?

When a crash happens, does the stack trace contains all the required information?

Reply:
> However, there is a risk that the build-time analysis of the application can cause failures at run time, due to not being able to reliably analyze various problematic code patterns (largely centered on reflection use). To mitigate these problems, warnings are produced whenever the trimmer cannot fully analyze a code pattern. For information on what the trim warnings mean and how to resolve them, see Introduction to trim warnings.

https://docs.microsoft.com/en-us/dotnet/core/deploying/trimming/trim-self-contained#:~:text=%20Publishing%20with%20Visual%20Studio%20%201%20On,to%20the%20platform%20you%20want%20to...%20More%20

Reply:
Update the project files to make trimming an opt-in experience with more comments about enabling trimming.